.github/workflows/main.yml

```diff
@@ -17,15 +17,15 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - run: npx @dappnode/dappnodesdk build --skip_save
-      
+
   e2e-test:
     runs-on: mainnet
     needs: build-test
     if: github.event_name != 'push'
     name: End to end tests
     steps:
       - uses: actions/checkout@v3
-      - run: npx @dappnode/dappnodesdk@latest github-action test-end-to-end --errorLogsTimeout 120 --healthCheckUrl http://beacon-validator.nimbus.dappnode:4500/eth/v1/beacon/genesis --network mainnet --environmentByService="{\"beacon-validator\":{\"CHECKPOINT_SYNC_URL\":\"https://checkpoint-sync.dappnode.io\"}}"
+      - run: npx @dappnode/dappnodesdk@latest github-action test-end-to-end --errorLogsTimeout 120 --healthCheckUrl http://beacon-validator.nimbus.dappnode:4500/eth/v1/beacon/genesis --network gnosis --environmentByService="{\"beacon-validator\":{\"CHECKPOINT_SYNC_URL\":\"https://checkpoint.gnosischain.com\"}}"
```

Comment:
I'm assuming that `http://beacon-validator.nimbus.dappnode:4500/eth/v1/beacon/genesis` has to be changed as well, as the Gnosis package won't have the same URL, right?

Reply:
You right, nice catch!